webapp/QAFs.md

```diff
@@ -0,0 +1,344 @@
+Quality Assured Features (QAFs)
+===============================
+
+Contents:
+- About this software
+- Compatibility
+- Features
+
+____________________________________________
+
+About this software
+-------------------
+
+- Name: Panoptes
+- Type: web application
+- Release: v2.0.beta4
+
+
+Compatibility
+-------------
+
+###Compatible server operating systems (and newer):
```

Comment:
If we take "and newer" off, then perhaps we could add a list of major versions for completeness / explicitly?
This was another attempt to circumvent the maintenance burden, by making the assumption that newer releases (of OSs, browsers) will remain compatible, since this is what we'd want. Usually we would expect older versions to be incompatible.
I agree that newer releases aren't always compatible, so we shouldn't assert it, but maybe we could soften the language and say something like "intended" instead? It reduces confusion, for example, if we can simply say: "Retwaq v3 and above on Tizen v2 and later should be OK" and note any known exceptions.